packages/vue-client/src/views/GameView.vue

```diff
@@ -197,15 +199,28 @@ const createTimeControlPreview = (
 
 <template>
   <div>
+
+    <!---
+    TODO: go board has some unnecessary top padding that should be removed
+    -->
+
     <component
       v-if="variantGameView && game_state?.state"
       v-bind:is="variantGameView"
       v-bind:gamestate="game_state.state"
       v-bind:config="config"
       v-on:move="makeMove"
     />
+
+    <div v-if="game.result" style="font-weight: bold; font-size: 24pt; margin-bottom: 0pt" >
```

Comment:
I believe these merge conflicts are what the CI is complaining about.  If you remove all the changes except the `DownloadSGF` component, it should build okay.

btw, the command is `yarn build` to build locally :)